src/integrationTest/java/uk/gov/hmcts/cp/cdk/http/AnswersHttpLiveTest.java

```diff
@@ -1,9 +1,10 @@
 package uk.gov.hmcts.cp.cdk.http;
 
-import static java.util.UUID.fromString;
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.*;
```

Comment:
please use CPP code formatter.. to not have imports like this.. 

use this https://github.com/CJSCommonPlatform/styleguide/blob/master/intellij-java-cjs-style.xml and also see this
https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=54952095#CodingGuidelines-Explicitimports